src/lib/journey/callbacks/_utilities/callback.utilities.test.ts

```diff
@@ -618,11 +619,11 @@ describe('Username Validation Failure message getter', () => {
         },
       ],
     });
-    const errorMessage = getPasswordValidationFailureText(
+    const errorMessage = getUsernameValidationFailureText(
```

Comment:
This is the test I refer to in my comment above, where we've mixed up `getPasswordValidationFailureText` and `getUsernameValidationFailureText`.